splunk-apm/README.md

```diff
@@ -5,70 +5,95 @@ Splunk OpenTelemetry Lambda Layer for AWS lambda enables seamless instrumentatio
+5. Splunk Metrics export
 
-   By default, SignalFx metrics are enabled with Splunk Metrics extension. Following configuration environment variables are required:
+   By default, Splunk metrics are enabled with Splunk Metrics extension. Following configuration environment variables are required:
    ```
    SPLUNK_REALM: <Splunk Realm>
    SPLUNK_ACCESS_TOKEN: <Access token>
    ```      
  
 ### Java wrapper
 
-The Java wrapper is based on OpenTelemetry Java version 1.5.0. 
+The Java wrapper is based on OpenTelemetry Java version 1.6.0. 
 
 The official documentation of the upstream layer can be found [here](https://github.com/open-telemetry/opentelemetry-lambda/blob/main/java/README.md).
 
-No code change is needed to instrument a lambda. However, the `AWS_LAMBDA_EXEC_WRAPPER` environment variable must be set accordingly: 
-- `/opt/otel-handler` for wrapping regular handlers implementing `RequestHandler`
-- `/opt/otel-proxy-handler` for wrapping regular handlers implementing `RequestHandler`, proxied through API Gateway, and enabling HTTP context propagation
-- `/opt/otel-stream-handler` for wrapping streaming handlers implementing `RequestStreamHandler`
+1. Installation
+
+    No code change is needed to instrument a lambda. However, the `AWS_LAMBDA_EXEC_WRAPPER` environment variable must be set accordingly: 
+    - `/opt/otel-handler` for wrapping regular handlers implementing `RequestHandler`
+    - `/opt/otel-proxy-handler` for wrapping regular handlers implementing `RequestHandler`, proxied through API Gateway, and enabling HTTP context propagation
+    - `/opt/otel-stream-handler` for wrapping streaming handlers implementing `RequestStreamHandler`
+
+2. Span flush
+
+    Spans are sent synchronously, before the lamda function terminates. Default span flush wait timeout is 1 second. It is controlled with a following environment variable (value in milliseconds):
+    ```
+    OTEL_INSTRUMENTATION_AWS_LAMBDA_FLUSH_TIMEOUT: 30000
+    ```
+   
+3. Context propagation
+
+    For more information about available context propagators, see the [Propagator settings](https://github.com/open-telemetry/opentelemetry-java/tree/v1.1.0/sdk-extensions/autoconfigure#customizing-the-opentelemetry-sdk) for the OpenTelemetry Java.
+
+4. Logging
+    
+    Logging is controller with `OTEL_LAMBDA_LOG_LEVEL` environment variable. Default set to `WARN`. When set to `DEBUG`, the logging exporter is added to traces exporter in order to help verify exported data.
+    
+    Please note that enabling `DEBUG` will generate additional logs which may in turn increase AWS CloudWatch costs. 
+
+### Python wrapper
+
+The Python wrapper is based on Splunk Distribution of OpenTelemetry Python version `v1.0.0`. 
+
+The official documentation of the upstream layer can be found [here](https://github.com/open-telemetry/opentelemetry-lambda/blob/main/python/README.md).
+
+1. Installation
+
+    Set the `AWS_LAMBDA_EXEC_WRAPPER` environment variable to `/opt/otel-instrument`.
```

Comment:
java uses `/opt/otel-handler`, should these be consistent between instrumentations? Or if the could clash then should they include language name to make clear which of these belongs to which language?

Reply:
That might make sense. Will open PR with upstream.